bitlum.go

```diff
@@ -0,0 +1,763 @@
+package client
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/shopspring/decimal"
+)
+
+const exchangeID = "bitlum"
+
+// Client is the http://exchange.bitlum.io exchange
+type Client struct {
+	backend
+}
+
+// NewExchangeClient create new bitlum exchange with specified URL and
+// authorization token.
+func NewClient(url string, authToken string) *Client {
+	return &Client{
+		backend: &graphqlBackend{
+			url:       url,
+			authToken: authToken,
+		},
+	}
+}
+
+// ID returns exchange ID
```

Comment:
What is exchange id from external user standpoint?

Reply:
This ID can be used to distinguish exchanges in zigzag. But in this client doesn't need. Will remove.